doc/getting-started/getting-started.rst

```diff
@@ -490,8 +480,9 @@ To generate a scenario configuration file and launch script:
       .. image:: ./images/gsg_config_launch_save.png
          :class: drop-shadow
 
-   #. Confirm that ``launch_uos_id3.sh`` appears in the directory
-      ``/home/<username>/acrn-work/my_board/output/``.
+   #. Confirm that ``launch_uos_id3.sh`` appears in the expected output directory::
+
+         ls ~/acrn-work/my_board/output/launch_uos_id3.sh
 
```

Comment:
Perhaps add a bit more detail as to why id3. Example: "Among the six User VMs, VM id3 is preconfigured to run a standard OS such as Ubuntu, while the other VMs have special uses."